odoo/cli/__init__.py

```diff
@@ -18,3 +18,5 @@
 from . import obfuscate
 from . import genproxytoken
 from . import db
+
+COMMAND = None
```

Comment:
You could put this inside `odoo.cli.command` because this init file is no longer there in master.

Reply:
I think we still have `odoo/cli/__init__.py' in master, no?

Reply:
My bad.